src/test/java/uk/gov/hmcts/reform/migration/service/MigrateChildrenServiceTest.java

```diff
@@ -0,0 +1,463 @@
+package uk.gov.hmcts.reform.migration.service;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.domain.common.CollectionEntry;
+import uk.gov.hmcts.reform.fpl.domain.CaseData;
+import uk.gov.hmcts.reform.fpl.domain.OldChild;
+import uk.gov.hmcts.reform.fpl.domain.OldChildren;
+import uk.gov.hmcts.reform.fpl.domain.Child;
+import uk.gov.hmcts.reform.domain.common.Address;
+import uk.gov.hmcts.reform.domain.common.EmailAddress;
+import uk.gov.hmcts.reform.domain.common.TelephoneNumber;
+import uk.gov.hmcts.reform.fpl.domain.common.ChildParty;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+class MigrateChildrenServiceTest {
```

Comment:
Really minor - should be called ChildrenDataMigrationServiceTest (if it's called like that Cmd Shift T shortcut doesn't work :()